compiler-rt/test/tsan/compare_exchange_acquire_fence.cpp

```diff
@@ -0,0 +1,46 @@
+// RUN: %clangxx_tsan -O1 %s -o %t && %run %t 2>&1 | FileCheck %s
```

Comment:
Exit code 0 should be enough
we probably do not need  | FileCheck here

Reply:
Done.

I also removed the CHECK and CHECK-NOT comment lines at the end of the file.